pkg/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -110,6 +112,14 @@ func NewNodePerformance(assetsDir string, profile *performancev2.PerformanceProf
 		templateArgs[templateGloballyDisableIrqLoadBalancing] = strconv.FormatBool(true)
 	}
 
+	splitClusterVersion := strings.Split(clusterVersion, ".")
```

Comment:
Can we use something nice to compare versions, for example, https://github.com/hashicorp/go-version will work for me.